newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -255,18 +250,18 @@ export const AssetDetails = ({
           </Column>
           <Column alignItems="center" justifyContent="center">
             <LeftLoader
-              isLoading={isBeingInstalled || (!asset && !error)}
+              isLoading={isBeingAddedToScene || (!asset && !error)}
               key="install"
             >
               <RaisedButton
-                primary
+                primary={!isAddedToScene}
                 label={
-                  isBeingInstalled ? (
-                    <Trans>Adding to my project...</Trans>
-                  ) : isAssetAdded ? (
-                    <Trans>Object added to my project</Trans>
+                  isBeingAddedToScene ? (
+                    <Trans>Adding...</Trans>
+                  ) : isAddedToScene ? (
+                    <Trans>Add again</Trans>
```

Comment:
Not sure if this is clear enough: Should it stay 'Add to the scene' and have a text below that reads "Already X in the scene"